cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -359,7 +362,55 @@ private void restartIfNecessary(int podId, RestartContext restartContext)
             } else {
                 throw e;
             }
+        } catch (IllegalArgumentException e) {
```

Comment:
The body of this catch clause seems to be identical to the `catch (ForceableProblem e)` above. Can we have just a single catch with the union type `catch (ForceableProblem | IllegalArgumentException e)`